components/placeholders/PostTextPlaceholder.tsx

```diff
@@ -0,0 +1,17 @@
+import { PostHeaderPlaceholder } from '@/components/placeholders/PostHeader';
+
+export const PostTextPlaceholder = ({ lines }: { lines: number }) => {
```

Comment:
we might want to define the props in a separate IProps interface as we did in other components. Just for consistency. What do you think of an default value in this component? would that make sense?

Reply:
we can add like 3 lines as per default 👍🏽